asynchronous-data-flow/sycl-2.2/03_interacting_with_data_on_the_host.md

```diff
@@ -153,11 +153,15 @@ Note that at least write access is required.
 ```cpp
 auto cgH = [=] (handler& h) {
   auto accA = bufA.get_access<access::mode::write>(h);
-  h.fill(accB, 10);
+  h.fill(accA, 10);
```

Comment:
Can you change the example to use a variable instead of a literal? just to make it a bit more ellaborate